storage/tests/unit/test_blob.py

```diff
@@ -564,6 +593,34 @@ def test__do_download_chunked(self):
             'GET', download_url, data=None, headers=headers)
         self.assertEqual(transport.request.mock_calls, [call, call])
 
+    def test__do_download_chunked_with_range(self):
+        blob_name = 'blob-name'
+        # Create a fake client/bucket and use them in the Blob() constructor.
+        client = mock.Mock(
+            _credentials=_make_credentials(), spec=['_credentials'])
+        bucket = _Bucket(client)
+        blob = self._make_one(blob_name, bucket=bucket)
+
+        # Modify the blob so there there will be 2 chunks of size 3.
```

Comment:
Is this going to be 4 chunks?

Reply:
as far as I understand there will be 2 chunks of size 2 (I changed that), why do you think there are 4 chunks?

Reply:
I thought there would be 4 chunks since it was bytes from 10-20 and the chunks were 3 bytes each.    Either way I think it makes sense now.  Thanks.